code/datums/api.dm

```diff
@@ -28,10 +28,10 @@
 		log_debug("API: Throttling bypassed - Command [command.name] set to no_throttle")
 	else
 		if(world_api_rate_limit[addr] != null && config.api_rate_limit_whitelist[addr] == null) //Check if the ip is in the rate limiting list and not in the whitelist
-			if(abs(world_api_rate_limit[addr] - world.time) < config.api_rate_limit) //Check the last request time of the ip
-				world_api_rate_limit[addr] = world.time // Set the time of the last request
+			if(abs(world_api_rate_limit[addr] - world.realtime) < config.api_rate_limit) //Check the last request time of the ip
```

Comment:
As said on discord, `realtime` is not precise enough (~52 second error) for the default value of api_rate_limit (5 seconds). `REALTIMEOFDAY` is likely what you want.